src/platform/site-wide/sass/modules/_m-medallia.scss

```diff
@@ -107,5 +107,5 @@
   top: 40px !important;
   right: 12px !important;
   z-index: 9999999;
-  color: #9f9fa4 !important;
+  color: $color-gray-light !important;
```

Comment:
![image](https://user-images.githubusercontent.com/2008881/119393184-771d1900-bc85-11eb-9b29-cdffc4170bff.png)
![image](https://user-images.githubusercontent.com/2008881/119393257-90be6080-bc85-11eb-9b71-ac79526ec396.png)

`color-gray-light` seemed like the closest color from https://design.va.gov/design/color-palette